bodhi/templates/override.html

```diff
@@ -76,6 +77,27 @@ <h2 class="pull-left">New Override</span>
         </div>
       </div>
 
+      <div class="col-md-3">
+        <div class="panel formpanel panel-default">
+          <div class="panel-heading clearfix">
+            <span class="pull-left">Submitter</span>
+          </div>
+          <div class="form-group noui text-left">
+            %if override is not UNDEFINED:
+            <a href="${request.route_url('user', name=override.submitter.name)}">
+              <img class="img-circle" src="${util.avatar(override.submitter.name, size=24)}"/>
+              ${override.submitter.name}
+            </a>
+            %else:
+            <a href="${request.route_url('user', name=request.user.name)}">
+              <img class="img-circle" src="${util.avatar(request.user.name, size=24)}"/>
+              ${request.user.name}
```

Comment:
Confused here too.  Oh, I think I see now!

- If the override is **not** UNDEFINED, then show the override submitter.
- If it *is* undefined, then show the logged in user.

I think this is because the override template here is used for both displaying existing overrides *and* for the form for creating new overrides.  So, if it's new ``<form>`` mode, then show *you* since you're going to be the one who ends up owning the override.